weave/trace_server/migrations/001_init.down.sql

```diff
@@ -1,8 +1,8 @@
 DROP TABLE call_parts;
 DROP TABLE calls_merged;
 
-DROP TABLE objects;
-DROP TABLE objects_deduped;
+DROP TABLE object_versions;
+DROP TABLE object_versions_deduped;
```

Comment:
Seems like `DROP TABLE` works but `DROP VIEW` would be more symmetric.
https://clickhouse.com/docs/en/sql-reference/statements/drop#drop-view

Reply:
Yep, thats nicer

Reply:
applied to all the `VIEW`s